### ISSUE TITLE
Callout: Focus does not go to the complete callout but only goes to the smaller controls inside the callout eg: button, links

### ISSUE THREAD

**Issue by anjali2408** [OfficeDev/office-ui-fabric-react#4941]
### Bug Report

- __Package version(s)__: 5.76.0

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable) Power BI Visio Visual

#### Describe the issue:

Callout: Focus does not go to the complete callout but only goes to the smaller controls inside the callout eg: button, links etc. Hence the narrator will never be able to read the text inside the callout unless we focus the text explicitly, which again creates confusion as if the text is actionable.
Setting the setInitialFocus to false also does not work and it focuses the first element that it finds inside the callout with a tabIndex >=0.

#### Actual behavior:

Callout: Focus does not go to the complete callout but only goes to the smaller controls inside the callout eg: button, links etc. setInitialFocus also is not functioning properly. even if it is false then also auto focus is set to some element with tabIndex >= 0

#### Expected behavior:

Callout: focus should go to the complete callout  when it is opened and narrator should be able to read the text withing the callout.

**Comment by dzearing** [comment]
I'm not totally sure this is true. You can have more than one callout on the page.

If you want to set focus to something within the callout, why not manage it within the content?
